.editorconfig

```diff
@@ -8,13 +8,11 @@ trim_trailing_whitespace = true
 
 [*.{py,yml}]
 indent_style = space
+max_line_length = 80
```

Comment:
Maybe 79 to be consistent with PEP 8?

Reply:
it just a ruller for the editor, stict check in the liter.